services/provider/api/interfaces/interfaces.go

```diff
@@ -9,12 +9,14 @@ type StorageClientStatus interface {
 	GetClusterID() string
 	GetClusterName() string
 	GetClientName() string
+	GetStorageQuotaUtilizationRatio() string
```

Comment:
Why a string? we want a ratio which means a floating point number

Reply:
The reason for choosing the string for this is i saw the platform version is string even when its floating so i made it string. Now i checked its values and saw it has multiple decimal points in it(like `4.16.0 / 4.17.0`). I will change to `float32`